hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -1859,7 +1859,7 @@ private FSDataOutputStream innerCreateFile(
         .withPutOptions(putOptions)
         .withIOStatisticsAggregator(
             IOStatisticsContext.getCurrentIOStatisticsContext().getAggregator())
-            .withMultipartAllowed(getConf().getBoolean(
+            .withMultipartEnabled(getConf().getBoolean(
```

Comment:
i think the multipart enabled flag should be made a field and stored during initialize(), so we can save on scanning the conf map every time a file is created.